BEGINNER_GUIDE.md

```diff
@@ -0,0 +1,167 @@
+
+---
+
+### Working on the issue
+Now since you have your first issue assigned, we will be discussing some basic steps that need to be followed while working on the issue.
+
+*Steps 1 to 3 are for setup purposes only. You have to follow these steps only once on each project.*
+
+*Steps 4 to 9 should be repeated for every new contribution.*
+
+
+**Step 1: Forking** 
+
+Fork the main project repository by clicking on the **Fork** button at the top-right.
+Forking will create a copy of the repository in your Github account.
+![fork](https://user-images.githubusercontent.com/64194946/93016931-570b6500-f5e2-11ea-98fe-f226934091e4.jpg)
+
+
+**Step 2: Cloning**
+
+*For further steps, you need to have git installed. You can download git from [here](https://git-scm.com/downloads)*
+
+Now since you have git installed, run the following command to clone the repository: 
+```
+git clone https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME
+```
+*where YOUR_GITHUB_USER_NAME is your Github user name and PROJECT_REPOSITORY_NAME is the name of the project repository.*
+
+
+**Step 3: Remote**
+
+First, navigate to your local repository using:
+```
+cd PROJECT_REPOSITORY_NAME
+``` 
+
+The default remote name of the local repository is *origin*. This remote points to the forked repository. But you need to keep track of the main project repository. For this, we need to create a new remote named *upstream*. 
+
+Run the following command to add new remote:
+```
+git remote add upstream https://github.com/anitab-org/PROJECT_REPOSITORY_NAME
+```
+
+Use `git remote -v` to check the current remote. It should show the following output:
+```
+$ git remote -v
+origin  https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME (fetch)
+origin  https://github.com/YOUR_GITHUB_USER_NAME/PROJECT_REPOSITORY_NAME (push)
+upstream        https://github.com/anitab-org/PROJECT_REPOSITORY_NAME.git (fetch)
+upstream        https://github.com/anitab-org/PROJECT_REPOSITORY_NAME.git (push)
+``` 
+
+
+**Step 4: Pull latest changes from upstream**
+
+It is pretty obvious that their might be some changes in the main project after you have forked it. Your local repository should be synchronous to the main repository.
+
+For this, you have to pull changes from upstream master branch to your local repository.
+
+Run the following command to pull the latest changes:
+```
+git pull upstream master
+```
+
+
+**Step 5: Creating new branch**
+
+It is a good practice to a create separate branch for every issue so that it is isolated from the *master* branch.
+
+Use the following command to create a new branch and switch to it:
+```
+git checkout -b BRANCH_NAME
+```
+
+*Try to choose appropriate BRANCH_NAME. I would recommend using issue number as branch name (as it is unique to every issue).*
```

Comment:
Remove content from I would... issue.
For example `beginnersGuide` name makes more sense than issue123
And if I have to look for  my changes later it will more easy if branch name is similar to issue task